### ISSUE TITLE
[BUG] Outdaded SrGB value calculation

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-windows#391]
**Describe the bug**
Per WCAG bug [Non-sRGB color spaces, outdated sRGB threshold, and other issues in the "relative luminance" definition](https://github.com/w3c/wcag/issues/360)

the number used to calculate relative luminance is incorrect. 

**Additional context**
Blocked until WCAG bug gets resolved.

**Comment by ferBonnin** [comment]
calculation of this is in [ColorContrastViewModel.cs](https://github.com/microsoft/accessibility-insights-windows/blob/master/src/AccessibilityInsights.SharedUx/ViewModels/ColorContrastViewModel.cs)

**Comment by Myndex** [comment]
I am the W3C member who is working on these issues in the WCAG. The correct threshold for the reverse transform (going to linear) is 0.04045 and this should be used, not 0.03928 as incorrectly listed in the WCAG Understanding document. Also, the sRGB working draft from 1996 is not the standard, IEC 61966-2-1:1999 is the relevant international standard for sRGB.

There is no real reason to use the incorrect threshold. In terms of the WCAG contrast measurement, it functionally makes no difference, but it can make a difference if you are using the equation for image processing. As a matter of best practices, the correct math should be used for converting sRGB to linear RGB, and this is the threshold value of 0.04045

For background, the reason for the change from the original sRGB draft has to do with a discontinuity at the intercept between the power curve and the linearized segment. 

There are some color transform libraries that use a more accurate (more decimal places) method which is advisable if you are doing multiple round trips for image processing applications. In the case of the reverse transform this is 0.0404482 (IEC rounded to 0.04045).  This is discussed to some degree in the [Wikipedia page on the subject.](https://en.wikipedia.org/wiki/SRGB)

Nevertheless the **_DEFINED STANDARD_** is **0.04045**

**Comment by DaveTryon** [comment]
This is fixed in https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.0926.01
